core/src/main/resources/jenkins/install/UpgradeWizard/footer.jelly

```diff
@@ -3,25 +3,52 @@
   <j:if test="${it.due}">
     <style>
       #upgrade-fragment {
-        background-color: #fce94f;
+        background: url(${rootURL}/images/jenkins.png) no-repeat 5% -63px;
+        padding: 10px;
+        box-shadow: 0 1px rgba(0,0,0,.1), 0 2px 5px rgba(0,0,0,.2), inset 0 -50px 50px -50px #cef;
+        position: relative;
       }
 
       #upgrade-notice {
-        font-weight: bold;
         text-align: center;
         font-size: 20px;
         height: 60px;
-
+        vertical-align:middle;
         display: flex;
         justify-content: center;
         align-items: center;
```

Comment:
It is, yes. The banner text is chopped into in-line blocks so as to get the button in there. We could do math to align the baseline of the text in the button to match the baseline of the text. We could use line-height, but vertical-align middle also does the job. Not sure what the harm is we might be concerned about.